test/locations.jl

```diff
@@ -10,5 +10,11 @@ using DataDeps: try_determine_load_path, determine_save_path, try_determine_pack
         @test try_determine_package_datadeps_dir(fn) == nothing
     end
 
-end
+    mktemp() do fn, fh
```

Comment:
I do not think this `mktemp()` is required

Reply:
You're right, again duplicated from above. Removed